floodviz/static/data/cities.json

```diff
@@ -0,0 +1,32 @@
+
```

Comment:
These are the cities given by Jordan in the "issues" tab on GitHub. I will modify the code so that they are contained in a variable instead.